modules/nf-core/tidk/explore/tests/main.nf.test

```diff
@@ -0,0 +1,86 @@
+nextflow_process {
+
+    name "Test Process TIDK_EXPLORE"
+    script "../main.nf"
+    process "TIDK_EXPLORE"
+    config "./nextflow.config"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "tidk"
+    tag "tidk/explore"
+
+    test("homo_sapiens-genome_21_fasta-success") {
+
+        when {
+            process {
+                """
+                input[0] = [
+                    [ id:'test' ], // meta map
+                    file(params.test_data['homo_sapiens']['genome']['genome_21_fasta'], checkIfExists: true)
+                ]
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(process.out).match() },
+                { assert snapshot(process.out.versions).match("versions") }
```

Comment:
Hi @leoisl 

Thanks for the feedback. Much appreciated.

This line ensures that the `versions` snapshot is constant across tests and configurations. In future if due to tool version bump the versioning snippet is changed in the script but not in the stub, this test will help catch that potential bug.

Reply:
Oh wow I haven't thought of that! Yes, it makes total sense! Now I want to change all my stub tests to do the same :joy: . Thanks for the explanation, I really appreciate it, this will make my testings more robust! Please feel free to merge whenever you want!